src/main.ts

```diff
@@ -90,6 +93,26 @@ function roll(val?: number) {
   charDescElem.textContent = getNameForByteChar(byte)
   hexElem.textContent = byte.toString(16).padStart(2, "0").toUpperCase()
 
+  const opcodeInfo = OPCODES[byte];
+
+  opcodeElem.appendChild(document.createElement("span")).textContent = opcodeInfo.name;
+
+  const descElem = document.createElement("span");
+  descElem.textContent = opcodeInfo.desc;
+  descElem.style.fontSize = "16px";
+  opcodeElem.appendChild(descElem);
+
+  const addressingElem = document.createElement("span");
+  addressingElem.textContent = opcodeInfo.addressing;
+  addressingElem.style.fontSize = "16px";
+  opcodeElem.appendChild(addressingElem);
```

Comment:
Can we shift the font-size here into the CSS?

Something like this should the opcode spans:
```css
#opcode>span:not(:first-child) {
  font-size: 16px;
}
```

Reply:
tested and looks good to me! updated the diff